unstructured/ingest/v2/processes/embedder.py

```diff
@@ -58,6 +58,14 @@ def get_embedder(self) -> BaseEmbeddingEncoder:
             )
 
             return VertexAIEmbeddingEncoder(config=VertexAIEmbeddingConfig(**kwargs))
+
+        elif self.embedding_provider == "voyage":
```

Comment:
Please change "voyage" to "langchain-voyageai"? You can refer to: https://github.com/Unstructured-IO/unstructured/blob/main/unstructured/ingest/interfaces.py#L237